app/models/concerns/profile_fields.rb

```diff
@@ -38,6 +38,24 @@ def canonical?
       parent_profile_id.blank?
     end
 
+    def rules_and_rule_groups
+      custom_arrange(relationships_for('conflicts'), relationships_for('requires'))
+    end
```

Comment:
@skateman btw, I couldn't put the body of `custom_arrange` in this method because it was too many lines with `relationships_for('requires')` and `relationships_for('conflicts')` calls :/